core/server/hooks.ts

```diff
@@ -66,7 +66,7 @@ const {
 const {
   hook: beforeOutputRenderedResponseHook,
   executor: beforeOutputRenderedResponseExecutor
-} = createMutatorHook<any, any>()
+} = createMutatorHook<any, string | { output: string }>();
```

Comment:
Please replace:
```ts
{ output: string }
```
With:
```ts
{ output: string, [key: string]: any }
```

Because returned object will be wider by default

Reply:
After I make the changes, should I create another pull request or can I do it re-using the existing one (#5242)? Sorry for the question, first-timer here.

Reply:
Pushing a new commit to this branch should automatically update this PR. Earlier, you had to create a new one to change a target branch.